examples/resources/blameless_incident_type/resource.tf

```diff
@@ -25,15 +25,26 @@ resource "blameless_incident_type" "incident_type_security" {
     ]
 
     # MS Teams Notifications
-    teams_invited_users = [
-      "@dillon"
-    ]
-    teams_announcement_groups = [
-      "@responders"
-    ]
-    teams_announcement_channels = [
-      "#general"
-    ]
+    teams_announcement_group = "fa1a4481-4ba9-45df-96b9-71c48163bd56"
```

Comment:
I'm not terribly familiar with MS Teams, but is this id easily available to normal users? Would it make more sense to take in a group name and translate it to the id in the API?

Reply:
This id can be retrieved from the MS Teams application.